variables.robot

```diff
@@ -201,6 +201,13 @@ ${OS_UBUNTU}=               ubuntu
 # MinnowBoard Turbot
 &{RTE55}=                   ip=192.168.10.112
 ...                         platform=minnowboard-turbot
+...                         sonoff_ip=192.168.4.35    pikvm_ip=192.168.4.180
```

Comment:
This PR may need a rebase after cleanup

Reply:
I must have missed this while rebasing. I removed the line 3464d520c5ad486c12f082facf9331d400173e64